### DISCUSSION TITLE
Speaker Identification Embeddings for Diarization

### DISCUSSION THREAD

**Discussion by pscotechadmin** [speechbrain/speechbrain#2319] [Q&A]
Has anybody successfully used the embeddings and a clustering algorithm for diarization? It seems like a decent approach but I am sure I am missing something.

**Comment by Tortoise17** [comment]
Is there any update or answer available for this??

**Reply by pscotechadmin** [reply]
Tortoise - if it helps I went with a paid service which was well worth the $$s

**Reply by Tortoise17** [reply]
@pscotechadmin and how much it costed you for a offline mode solution?

**Reply by pscotechadmin** [reply]
~$5-10 per year per voice 'fingerprint' with a minimum fee to get going
